components/nav/Header.vue

```diff
@@ -359,9 +359,9 @@ export default {
           <template slot="popover" class="user-menu">
             <ul class="list-unstyled dropdown" @click.stop="showPageActionsMenu(false)">
               <li v-for="a in pageActions" :key="a.label" class="user-menu-item">
-                <a v-if="!a.seperator" @click="pageAction(a)">{{ a.labelKey ? t(a.labelKey) : a.label }}</a>
-                <div v-else class="menu-seperator">
-                  <div class="menu-seperator-line" />
+                <a v-if="!a.separator" @click="pageAction(a)">{{ a.labelKey ? t(a.labelKey) : a.label }}</a>
```

Comment:
Possibly an API change?

Reply:
This is only used in one place (on the home page action menu in the header) and the same change was made to the config there